bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/SimpleRow.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.grpc;
+
+import com.google.bigtable.v2.Cell;
+import com.google.bigtable.v2.Column;
+import com.google.bigtable.v2.Family;
+import com.google.bigtable.v2.Row;
+import com.google.protobuf.ByteString;
+
+import java.util.ArrayDeque;
+
+public class SimpleRow {
+  public class SimpleColumn {
+    public byte[] family;
+    public ByteString qualifier;
+    public Column.Builder builder;
+  }
+  private final ArrayDeque<SimpleColumn> columns;
+	
+  public SimpleRow() {
+    columns = new ArrayDeque<SimpleColumn>();
+  }
+	
+  public void addCell(String family, ByteString qualifier, Cell cell) {
+    SimpleColumn column = new SimpleColumn();
+    column.family = family.getBytes();
```

Comment:
I don't think you need to turn this into bytes and then back into a String. Can it be left a String and compared against previous families using .equals() instead of the byte array comparison?